### PR TITLE
Max out the unread count at 99.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk 

img_0028

img_0027

### CODE REVIEW

**Comment by charlesmchen** [comment]
Those screenshots are "before", demonstrating the issue this PR intends to address.

File: Signal/src/ViewControllers/ConversationView/MessagesViewController.m [hunk 1]
```diff
@@ -3812,7 +3812,8 @@ - (void)clearDraft
 - (void)updateBackButtonUnreadCount
 {
     AssertIsOnMainThread();
-    self.backButtonUnreadCount = [self.messagesManager unreadMessagesCountExcept:self.thread];
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/ConversationView/MessagesViewController.m]: Can we fit in a "99+"? Or would that look bad.

Comment (charlesmchen) [inline Signal/src/ViewControllers/ConversationView/MessagesViewController.m]: Good question.

Comment (charlesmchen) [inline Signal/src/ViewControllers/ConversationView/MessagesViewController.m]: I tried, it doesn't fit.

**Pull request merged**